src/main/java/org/gridsuite/network/map/dto/vscconverterstation/VscConverterStationTabInfos.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.network.map.dto.vscconverterstation;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.powsybl.iidm.network.Identifiable;
+import com.powsybl.iidm.network.Terminal;
+import com.powsybl.iidm.network.VscConverterStation;
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+/**
```

Comment:
Esthetical : could you lift the empty line above your the javadoc block, here and some other files ?

Reply:
Even if it's checkstyle validated?

Reply:
Checkstyle has limits too ;-)

Reply:
Done